.travis.yml

```diff
@@ -42,7 +42,7 @@ install:
      tesseract-ocr-deu
      unzip
   - mkdir -p ~/.m2 ~/bin
-  - wget -q https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-64bit-static.tar.xz
+  - wget -q https://pkg.opencast.org/bin/ffmpeg-release-64bit-static.tar.xz
```

Comment:
... results in 403:

```
wget https://pkg.opencast.org/bin/ffmpeg-release-64bit-static.tar.xz
--2018-05-16 13:30:31--  https://pkg.opencast.org/bin/ffmpeg-release-64bit-static.tar.xz
Aufl"osen des Hostnamens pkg.opencast.org (pkg.opencast.org)... 131.173.22.143
Verbindungsaufbau zu pkg.opencast.org (pkg.opencast.org)|131.173.22.143|:443 ... verbunden.
HTTP-Anforderung gesendet, auf Antwort wird gewartet ... 403 Forbidden
2018-05-16 13:30:32 FEHLER 403: Forbidden.
```

Reply:
It should do that for you ;-P
Only Travis is allowed to download that freely

Reply:
If you look at the [logs on Travis](https://travis-ci.org/opencast/opencast/jobs/379670275) you can see that this point has already passed successfully.

Reply:
Thanks, I'll proceed merging.

Reply:
Thanks, @staubesv, for the swift review!